WordPress/Classes/Models/Blog.m

```diff
@@ -439,6 +444,46 @@ - (BOOL)supports:(BlogFeature)feature
     }
 }
 
+- (BOOL)supportsPublicize
+{
+    // wpcom blog and publicize has not been disabled
+    if (self.isHostedAtWPcom && ![[self getOptionValue:OptionsKeyPublicizeDisabled] boolValue]) {
+        return YES;
+    }
+
+    // self-hosted connected to jetpack via the wpcom acct signed into the app.
+    if (self.dotComID && [self jetpackPublicizeModuleEnabled]) {
+        return YES;
+    }
```

Comment:
`jetpackRESTSupported` returns `NO` in this case as `self.jetpackAccount` is nil.